.github/workflows/integration-test-trigger.yml

```
@@ -21,11 +23,17 @@ jobs:
       # The owner of the provided token must have write access to trigger this workflow run.
       - name: Trigger the real workflow
         run: |
+          if [ ${{ github.event_name }} == "pull_request" ]; then
```

Most of these contexts have an equivalent default environment variable, and I generally find it safer to use the variable when interpolated into Bash.
```suggestion
          if [ "$GITHUB_EVENT_NAME" == "pull_request" ]; then
```